trackintel/analysis/location_identification.py

```diff
@@ -0,0 +1,214 @@
+import numpy as np
+import pandas as pd
+
+
+def location_identifier(sps, pre_filter=True, recipe="FREQ"):
+    """Find home and work/location for each user with different recipes.
+
+    Parameters
+    ----------
+    sps : Geodataframe (as trackintel staypoints)
+        Staypoints with column "location_id".
+
+    pre_filter : bool, default True
+        Prefiltering the staypoints to exclude locations with not enough data.
+        The filter function can also be accessed via `pre_filter`.
+
+    recipe : {'FREQ'}, default "FREQ"
+        Choose which recipe to use.
+        - FREQ: Select the most visited location
+        - PASS: just for decoration.
+
+    Returns
+    -------
+    Geodataframe (as trackintel staypoints)
+        With additional column assigning one of three location identies {'home', 'work', None}.
```

Comment:
might be good to state the name of the additional column. I think it's `activity_label`?